pynemo/pynemo_exe.py

```diff
@@ -5,7 +5,7 @@
 '''
 
 import sys, getopt
-from . import profile

```

Comment:
Apparently this is not so good in py3. Also it breaks my PyCharm debugging session, which is well worth having. Bye bye print statements.

Reply:
It turns out `import profiler` breaks the command line execution. So I've restored that. However keeping `#import profiler` because debugging with PyCharm is a game changer.

Reply:
why is:
`from . import profiler`
bad in Py3?

Reply:
Hmm. So maybe it isn't. I read it somewhere when trying to figure out why this syntax wouldn't work in my PyCharm console. But making a change broke the command line execution. So this is restored to ``from . import profiler``, but I make local changes to use PyCharm debug sessions.